examples/deployment-oss-min/README.md

```diff
@@ -52,4 +53,21 @@ For more information about how to configure the NGINX Ingress Controller, check
 1. Delete the namespace:
     ```
     kubectl delete namespace my-nginx-ingress
-    ```
\ No newline at end of file
+    ```
+   
+## TransportServers
+
+TransportServers provide support for TCP/UDP but are in active development and provided as a preview feature. 
+A GlobalConfig resource is used to specify the TCP/UDP listeners and is required by TransportServers. 
+To use TransportServers, you must create a GlobalConfig resource *before* starting the Operator.
```

Comment:
should we specify that it has to be created after namespace(s)?

Reply:
updated